ci/check-golden-md5-hashes.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+source "$(dirname "$0")/lib/init.sh"
+source module lib/io.sh
+
+if [[ "${CHECK_GENERATED_CODE_HASH}" != "yes" ]]; then
+  echo "Skipping the generated code hash check as it is disabled for this build."
+  exit 0
+fi
+
+cd "${PROJECT_ROOT}"
```

Comment:
Do we need the `${PROJECT_ROOT}/` prefixes below if we're going to `cd` (or vice versa)?